cluster/src/test/java/com/hartwig/pipeline/testsupport/TestInputs.java

```diff
@@ -192,6 +195,8 @@ public static SageOutput sageGermlineOutput() {
                 .status(PipelineStatus.SUCCESS)
                 .maybeFinalVcf(gsLocation(somaticBucket(SageGermlineCaller.NAMESPACE),
                         RESULTS + TUMOR_SAMPLE + ".germline." + FileTypes.GZIPPED_VCF))
+                .maybeGermlineGeneCoverageTsv(gsLocation(somaticBucket(SageGermlineCaller.NAMESPACE),
+                        RESULTS + TUMOR_SAMPLE + SageGermlineCaller.SAGE_GENE_COVERAGE_TSV))
```

Comment:
So are these the reason for the `public` in the constants? Feels a bit backwards to me, making the implementation constants `public` but then also adding `private` constants in the tests. Maybe it's not so bad but it definitely breaks encapsulation...